query_select.go

```diff
@@ -159,6 +160,13 @@ func (q *SelectQuery) WhereAllWithDeleted() *SelectQuery {
 
 //------------------------------------------------------------------------------
 
+func (q *SelectQuery) IndexHints(indexes ...string) *SelectQuery {
```

Comment:
Let's rename this to `UseIndex`, because there is also `ForceIndex` and `IgnoreIndex`